core/src/test/scala/clue/ErrorPolicySpec.scala

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package clue
+
+import cats.data.Ior
+import cats.data.NonEmptyList
+import clue.model.GraphQLError
+import clue.model.GraphQLResponse
+import org.scalacheck.Arbitrary
+import org.typelevel.log4cats.testing.StructuredTestingLogger
+import cats.effect.IO
+import cats.syntax.all.*
+
+class ErrorPolicySpec extends munit.CatsEffectSuite {
```

Comment:
Awesome that you added tests!

Reply:
I was surprised I changed some logic and no tests failed!